components/time-picker/time-picker.component.ts

```diff
@@ -281,6 +285,11 @@ export class NzTimePickerComponent implements ControlValueAccessor, OnInit, Afte
   ngOnInit(): void {
     this.inputSize = Math.max(8, this.nzFormat.length) + 2;
     this.origin = new CdkOverlayOrigin(this.element);
+
+    this.i18nPlaceHolder$ = this.i18n.localeChange.pipe(
+      map((nzLocale: NzI18nInterface) => nzLocale.TimePicker.placeholder),
+      startWith(this.i18n.getLocaleData(`TimePicker.lang.placeholder`))
+    );
```

Comment:
```suggestion
    this.i18nPlaceHolder$ = this.i18n.localeChange.pipe(
      map((nzLocale: NzI18nInterface) => nzLocale.TimePicker.placeholder)
    );
```
Already has the initial value, no need to use `startWith`.